selectionSort.js

```diff
@@ -1,3 +1,4 @@
+// selection sort algorithm
 function selectionSort(arr) {
```

Comment:
Insert newline above the outer for loop
Insert newline above the inner for loop
Insert newline above the `const temp ...`